README.md

```diff
@@ -50,6 +50,12 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
 * To disable fedora internal audit capability, edit `install_scripts/config` and change the FEDORA_AUDIT variable to false. The FEDORA_AUDIT_LOCATION can also be changed from its default "/audit", if necessary.
 * `vagrant up`
 
+####Acceptance Testing
+
+One may validate the state of the environment provisioned by the BASH scripts using [beaker](https://github.com/puppetlabs/beaker).
+The acceptance tests (implemented using [server-spec](http://serverspec.org/)) may be executed locally in the following manner:
+* `rake beaker`
```

Comment:
> When you say "executed locally" in which directory do you mean? Ideally, this does not add any host-machine requirements

My apologies, this refers to the local environment into which the GitHub repository was cloned.  It should be rephrased to explicitly state that the tests are executed *outside* of the VM instantiated.